### PR TITLE
WIP – community page updates

### PR DESCRIPTION
**Pull request by gr2m**
# This is work in progress – do not merge yet

before merging, get okay from
- [x] @Charlotteis 
- [x] @jsimplicio

### CODE REVIEW

File: community/index.html [hunk 1]
```diff
@@ -152,6 +170,24 @@ <h1 class="h3">
...
+            <span class="h6">UX Developer / Designer</span>
+            <h1 class="h3">
+              <a name="julia-schmidt" class="jump-mark">Julia Simplicio</a>
+            </h1>
+            <p>
+                Design-focused developer helping to bring more designers to Open Source.
+            </p>
+        </article>
+        <aside class="bio-link">
+            <a href="https://github.com/jsimplicio" target="_blank">Twitter</a> | <a href="https://github.com/jsimplicio" target="_blank">GitHub</a>
```
Comment (NickColley) [inline community/index.html]: Not sure Julia has a twitter, maybe we could link to their website http://juliasimplicio.com/

Comment (gr2m) [inline community/index.html]: oops, her twitter is https://twitter.com/jlsmp :)

File: community/index.html [hunk 2]
```diff
@@ -82,6 +82,24 @@ <h1 class="h3">
...
+            <span class="h6">Developer &amp; Community Person</span>
+            <h1 class="h3">
+              <a name="carolina-buzio" class="jump-mark">Charlotte Spencer</a>
+            </h1>
+            <p>
+                Working on making technology more accessible. JavaScript and Emoji enthusiast. Creator of <a href="https://yourfirstpr.github.io/" target="_blank">Your First PR</a>.
+            </p>
+        </article>
+        <aside class="bio-link">
+            <a href="http://twitter.com/charlotteis" target="_blank">Twitter</a> | <a href="http://github.com/charlotteis" target="_blank">GitHub</a>
```
Comment (varjmes) [inline community/index.html]: change to https twitter & github links? :)

File: community/index.html [hunk 3]
```diff
@@ -82,6 +82,24 @@ <h1 class="h3">
     <div class="cb person-bio">
         <article>
             <figure class="grid-3 l">
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/charlotte-spencer.jpg" width="152" height="152" alt="Portrait photo(black and white) of Charlotte Spencer." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/charlotte-spencer--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Charlotte Spencer." />
+            </figure>
+            <span class="h6">Developer &amp; Community Person</span>
+            <h1 class="h3">
+              <a name="carolina-buzio" class="jump-mark">Charlotte Spencer</a>
```
Comment (varjmes) [inline community/index.html]: My name is not carolina-buzio :D (thats what you gave the `<a name=>` as)

File: community/index.html [hunk 4]
```diff
@@ -152,6 +170,24 @@ <h1 class="h3">
...
     </div>
     <div class="cb person-bio">
+        <article>
+            <figure class="grid-3 l">
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/julia-simplicio.jpg" width="152" height="152" alt="Portrait photo(black and white) of Julia Simplicio." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/julia-simplicio--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Julia Simplicio." />
+            </figure>
+            <span class="h6">UX Developer / Designer</span>
+            <h1 class="h3">
+              <a name="julia-schmidt" class="jump-mark">Julia Simplicio</a>
```
Comment (varjmes) [inline community/index.html]: Need to change the `<a name=>` to Julias name. Right now the link is named "julia-schmidt".

File: community/index.html (hunk 3 above)
Comment (nickcolley) [inline community/index.html]: Also in HTML5 they should be `id` instead of `name`

**Comment by gr2m** [comment]
Thanks a lot for the review @Charlotteis!

**Comment by varjmes** [comment]
LGTM :)

**Comment by gr2m** [comment]
I’ve rebased commits on `gh-pages`. Can be merged when Julia gives her ok

**Pull request merged**